tgui/packages/tgui/interfaces/JobSelection.tsx

```diff
@@ -90,11 +95,8 @@ export const JobEntry: SFC<{ job; department; act: Function }> = (data) => {
 
 export const JobSelection = (props, context) => {
   const { act, data } = useBackend<Data>(context);
-  const { departments } = data;
-
-  const anyJobAvailable = departments.filter((department) =>
-    department.jobs.some((job) => !job.unavailable_reason)
-  );
+  let departments = JSON.parse(JSON.stringify(data.departments)); // Why the fuck is it so hard to clone objects properly in JS?!
```

Comment:
No.

Reply:
Yea, that doesn't help.

I've had a look, and I've not come across a solution that actually makes deep copies, cause you guys want data to be treated as immutable.